Samples/Windowing/cpp-winui/SampleApp/packages.config

```diff
@@ -2,5 +2,5 @@
 <packages>
   <package id="Microsoft.Windows.CppWinRT" version="2.0.210722.2" targetFramework="native" />
   <package id="Microsoft.Windows.SDK.BuildTools" version="10.0.22000.194" targetFramework="native" />
-  <package id="Microsoft.WindowsAppSDK" version="1.0.0" targetFramework="native" />
+  <package id="Microsoft.WindowsAppSDK" version="1.1.0-preview2" targetFramework="native" />
```

Comment:
This should be "1.1.0" not targeting the preview2 release.
Note: this is perhaps not blocking for this PR, but needs to be updated once the 1.1.0 release is live.